### PR TITLE
Add additional details around healthcheck operation for TFE

### PR DESCRIPTION
**Pull request by hashitop**

### CODE REVIEW

File: content/source/docs/enterprise/admin/monitoring.html.md [hunk 1]
```diff
@@ -11,6 +11,17 @@ This document outlines best practices for monitoring a Terraform Enterprise inst
...
 Terraform Enterprise provides a `/_health_check` endpoint on the instance. If Terraform Enterprise is up, the health check will return a `200 OK`.
 
+The `/_health_check` endpoint operates in 2 modes
+
+- Full check
+- Minimal check
+
+With full check, the service will attempt to verify status of internal component  and PostgreSQL, in contrast to minimal check in which only returns `200 OK`.
+
+In normal circumstance, full check will be performed during start up of the instance, once Terraform Enterprise started up in active running state, any subsequent checks will be performed in minimal check mode.
```
Comment (evanphx) [inline content/source/docs/enterprise/admin/monitoring.html.md]:
```suggestion
In normal circumstances, a full check will be performed during the start of an instance and once Terraform Enterprise is in an active running state, any subsequent checks will perform a minimal check.
```

File: content/source/docs/enterprise/admin/monitoring.html.md [hunk 2]
```diff
@@ -11,6 +11,17 @@ This document outlines best practices for monitoring a Terraform Enterprise inst
...
+
+-> **Note:** If you wish to perform full check forcefully, additional query parameter is required with `/_health_check?full=1` and extra caution as every single call will make contact to internal component and PostgreSQL.
```
Comment (evanphx) [inline content/source/docs/enterprise/admin/monitoring.html.md]:
```suggestion
-> **Note:** If you wish to force a full check, an additional query parameter is required: `/_health_check?full=1`. Take extra caution as every call will make contact internal components and PostgreSQL, increasing system load and latency.
```

File: content/source/docs/enterprise/admin/monitoring.html.md [hunk 3]
```diff
@@ -11,6 +11,17 @@ This document outlines best practices for monitoring a Terraform Enterprise inst
 
 Terraform Enterprise provides a `/_health_check` endpoint on the instance. If Terraform Enterprise is up, the health check will return a `200 OK`.
 
+The `/_health_check` endpoint operates in 2 modes
```
Comment (evanphx) [inline content/source/docs/enterprise/admin/monitoring.html.md]:
```suggestion
The `/_health_check` endpoint operates in 2 modes:
```

File: content/source/docs/enterprise/admin/monitoring.html.md (hunk 1 above)
Comment (nfagerlund) [reply content/source/docs/enterprise/admin/monitoring.html.md]:
```suggestion
The endpoint's default behavior is to perform a full check during startup of the instance, and minimal checks after Terraform Enterprise is active and running.
```

File: content/source/docs/enterprise/admin/monitoring.html.md (hunk 2 above)
Comment (nfagerlund) [reply content/source/docs/enterprise/admin/monitoring.html.md]:
```suggestion
-> **Note:** If you wish to force a full check, an additional query parameter is required: `/_health_check?full=1`. Take extra caution as every call will make requests to internal components and PostgreSQL, increasing system load and latency.
```

**Comment by nfagerlund** [comment]
@hashitop Hello! If you're willing to merge the suggestions on this, we can get it merged. Since it's from your fork of the repo, I can't just commit them myself.

**Comment by nfagerlund** [comment]
Looks like at some point I gained the power of "let maintainers edit this pr" or something. I'm shippin it.

**Pull request merged**